### DISCUSSION TITLE
How to use the `show` rule to set the color of all Cetz object?

### DISCUSSION THREAD

**Discussion by tapyu** [cetz-package/cetz#630] [Q&A] [answered]
I want to set the color of Cetz objects (such as `line`, `circle`, `content`, ...) to red.

The problem is that they aren't an [element](https://github.com/typst/typst/issues/662). I wonder if they share a common element so that a `show` rule be applied to it can affect all of them.

**Comment by jneug** [comment, marked as answer]
Dou don't use `#show` here, but rather the cetz function `set-style`:

```typst
#cetz.canvas({
  import cetz.draw: *

  set-style(
    fill: red,
    stroke: red
  )

  line((0,0), (5,0))
  circle((), radius:1)
})
```

**Reply by noahjutz** [reply]
Is there a cetz alternative to show, for example in order to draw a specific shape on the coordinates of every `cetz.content`?

**Reply by fenjalien** [reply]
No, you could create your own function in the canvas scope to do that though:
```
let show-content(pt, ..args) = {
  content(pt, ..args)
  circle(())
}
```
